test/unit/services/oooRequest.test.ts

```
@@ -0,0 +1,180 @@
+import { validOooStatusRequests } from "../../fixtures/oooRequest/oooRequest";
+
+const requestModel = firestore.collection("requests");
+
+describe("Test OOO Request Service", function() {
+    let validOOORequest;
+    let testUserStatus;
+    const testReason = "Due to some personal reason";
+    const testUserId = "11111";
+    const errorMessage = "Unexpected error occured";
+
+    afterEach(async function() {
```

ESLint

[error] Replace `··afterEach(async·function` with `afterEach(async·function·`

(prettier/prettier)